src/Catty/DataModel/Formula/FormulaElement.h

```diff
@@ -72,19 +72,22 @@ typedef NS_ENUM(NSInteger, IdempotenceState) {
 @property (nonatomic, strong) NSString* value;
 @property (nonatomic, strong) FormulaElement* leftChild;
 @property (nonatomic, strong) FormulaElement* rightChild;
+@property (nonatomic, strong) FormulaElement* additionalChildren;
 @property (nonatomic, strong) FormulaElement* parent;
 @property (nonatomic) IdempotenceState idempotenceState;
 
 - (id)initWithType:(NSString*)type
```

Comment:
Suggestion: instead of passing `nil` at almost all places, create a new initializer and use it only at those places where it is necessary. The old initializer could simple forward to this initializer while passing `nil` internally, instead of passing it at every call.